src/simulation.h

```diff
@@ -17,22 +22,22 @@ typedef struct chunk {
 // always freed as part of a simulation history
 Chunk *new_chunk();
 
-typedef struct simulationHistory {
+typedef struct simulation_history {
```

Comment:
For consistency, it would be nice to keep this as `simulationHistory`. 

I usually do the following:
- underscores for variables and functions
- camelCase for struct ids
- CamelCase for struct typedefs